src/component/menu/__tests__/routes-test.jsx

```diff
@@ -1,12 +1,12 @@
 import { routes, baseRoutes, getRoute } from '../routes';
 
 test('returns all defined routes', () => {
-    expect(routes.length).toEqual(32);
+    expect(routes.length).toEqual(33);
     expect(routes).toMatchSnapshot();
 });
 
 test('returns all baseRoutes', () => {
-    expect(baseRoutes.length).toEqual(10);
+    expect(baseRoutes.length).toEqual(11);
```

Comment:
Not relevant for this PR: 
it's kind of stupid that we have these test, verifying we have managed to put something inside an array. We should probably instead verify that we have critical baseRoutes still in place (e.g. /features, /strategies etc).